examples/safety_and_security/retail_agent/README.md

```
@@ -0,0 +1,819 @@
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# NeMo Agent Safety and Security Engine (NASSE)
+
+### Demonstrated Through Retail Agent Example
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)